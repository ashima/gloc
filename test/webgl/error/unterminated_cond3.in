#version 100
#ifdef FOO
