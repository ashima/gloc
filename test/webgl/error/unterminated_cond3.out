File 0, line 2, col 0 - 6:
unterminated conditional "#ifdef"

Fatal: unrecoverable parse error (2)
